src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -135,6 +135,15 @@ namespace OpenLoco::Ui::PromptBrowse
     static void sub_446E87(window* self);
     static bool filenameContainsInvalidChars();
 
+    // 0x004CEB67
+    // TODO: replace with Ui::TextInput::calculateTextOffset
+    static void calculateTextOffset(int16_t dx)
```

Comment:
why can't we do this atm

Reply:
I wrote an explanation in the commit message:
> InputSession::calculateTextOffset is using its own (integrated) variables, instead of the loco_global ones still used by PromptBrowseWindow. We therefore temporarily revert to using the original routine in PromptBrowseWindow, until the latter has been refactored to use an InputSession.

 It's actually already done in #841.

Reply:
Okay cool thought it would be something like that.